packages/in_app_purchase/example/tizen/tizen-manifest.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="org.tizen.in_app_purchase_tizen_example" version="1.0.0" api-version="5.5" xmlns="http://tizen.org/ns/packages">
+    <profile name="tv"/>
+    <ui-application appid="org.tizen.in_app_purchase_tizen_example" exec="Runner.dll" type="dotnet" multiple="false" nodisplay="false" taskmanage="true">
+        <label>in_app_purchase_tizen_example</label>
+        <icon>ic_launcher.png</icon>
+        <metadata key="http://tizen.org/metadata/prefer_dotnet_aot" value="true"/>
+    </ui-application>
+    <feature name="http://tizen.org/feature/screen.size.all"/>
+    <privileges>
+        <privilege>http://developer.samsung.com/privilege/billing</privilege>
+        <privilege>http://developer.samsung.com/privilege/sso.partner</privilege>
```

Comment:
Is this privilege (`sso.partner`) mandatory? It's not listed in the privilege list in README.md.

If this is a partner-level privilege and requires a partner-level certificate, it should be documented properly (see flutter_webrtc_tizen or video_player_videohole).

Reply:
`sso.partner` is used for calling `sso_get_login_info` to get customId.
Fixed.

Reply:
Thanks. I added some additional information and removed redundant information.